src/cards/promo/Philares.ts

```
@@ -91,19 +91,23 @@ export class Philares implements CorporationCard {
     }
 
     public onTilePlaced(player: Player, space: ISpace, game: Game): void {
+      const philaresPlayer = game.getPlayers().find((player) => player.isCorporation(CardName.PHILARES));
+      if (philaresPlayer === undefined) {
+        console.error('Could not find Philares player');
```

Should we add a test for Philares player not found?

---

It should not happen though. I mean, if Philares is in play, it should have a player.
I mostly added the check so that TS doesn't complain about using `philaresPlayer.id` afterwards.

---

We would have failed silently prior to this change as all of the checks below would have never had a match. So the logging to `console.error` should be a net improvement.